src/app/src/setupProxy.js

```diff
@@ -0,0 +1,24 @@
+const proxy = require('http-proxy-middleware');
+const _ = require('lodash');
+
+const pathsToProxy = Object.freeze([
+    '/api',
+    '/web',
+    '/api-auth',
+    '/rest-auth',
+    '/user-login',
+    '/user-logout',
+    '/user-signup',
+    '/user-profile',
+    '/api-token-auth',
+    '/api-feature-flags',
+]);
+
+const djangoProxyTarget = Object.freeze({ target: 'http://django:8081' });
+
+const createProxies = app => _.forEach(
+    pathsToProxy,
+    path => app.use(proxy(path, djangoProxyTarget)),
+);
+
+module.exports = createProxies;
```

Comment:
I see statements like

```
[HPM] Proxy created: /api  ->  http://django:8081
```

being printed to my console, but I don't see in this PR where `createProxies` is being called. How is this wired in to the dev server?

Reply:
If a `src/setupProxies.js` file exists, the default export gets called as a function by Create React App: https://facebook.github.io/create-react-app/docs/proxying-api-requests-in-development#configuring-the-proxy-manually

Reply:
👍 Thanks for providing that pointer to the documentation.